### DISCUSSION TITLE
Agenda should skip to current day-of-week

### DISCUSSION THREAD

**Discussion by jaredmauch** [ietf-tools/datatracker#3742] [Ideas]
When visiting a meeting agenda page, based on the current time the visual element of the page should be sent directly to the current day or session time

**Comment by rjsparks** [comment]
That would violate the principle of least surprise, _and_ it would be hard to calculate where on the page to jump to - note that "now" applies to the day long secretariat related things as well as the current session - see where all the red bars are.

This is related to conversations about defaulting to browser time (counter argument has been that on-site networks often start with browser time in the wrong timezone)

**Reply by jaredmauch** [reply]
Disagree the least surprise, the server knows what time it is, and having the page fully render but move to the current_time() is likely why someone is visiting the page during the meeting, to find the links for the current session.  The page is super dense for a full week and on a Thursday/Friday someone can easily scroll up.

**Reply by rjsparks** [reply]
(This meeting is pretty much over, so not much utility in the observation, but it should be taken into account)
Does https://datatracker.ietf.org/meeting/agenda#now already do what you're asking?

**Comment by larseggert** [comment]
One option to reduce page complexity would be to do per-day content tabs (and then maybe have the current day active w/o scrolling around in the page.)

**Reply by rjsparks** [reply]
Would "search on page" do the right thing for someone trying to find a particular session?
